wrappers/README.md

```diff
@@ -12,9 +12,9 @@ This wrapper does not implement any extra functionality.
 
 ```hcl
 terraform {
-  source = "tfr:///terraform-aws-modules/sqs/aws//wrappers"
```

Comment:
It looks like the changes to this file are causing CI to fail - please revert these changes

Reply:
OK, those came from running the pre-commit container - not even sure what the function of the wrapper is. Anyways, I reverted changes in this file and force pushed a revised commit.